src/basics/building-deploying-interfacing.md

```diff
@@ -0,0 +1,133 @@
+
+_If you are experiencing issues with this step, we recommend returning to review the [requirements] page or hopping in our [Discord dev-chat] channel for support._
+
+### Deploy
+In this step, we will be submitting a transaction to store our program's logic on the Arch Network.
+
+```bash
+arch-cli deploy
+```
+
+Output:
+```bash
+Welcome to the Arch Network CLI
+  → Loading configuration from /Users/jr/Library/Application Support/arch-cli/config.toml
+Deploying your Arch Network app...
+  ℹ Building program...
+  ✓ Program built successfully
+  ℹ Program ID: 3688ef8de06d56e32a765243e900875c4fefc6aa9c83dfbc2643f661c5b4982e
+  ℹ Failed to load wallet 'devwallet'. Error: JSON-RPC error: RPC error response: RpcError { code: -18, message: "Wallet file verification failed. Failed to load database path '/home/bitcoin/.bitcoin/regtest/wallets/devwallet'. Path does not exist.", data: None }
+Attempting to resolve the issue...
+  ✓ Wallet 'devwallet' created successfully.
+  → Generating initial blocks for mining rewards...
+  ✓ Initial blocks generated
+  ✓ Transaction sent: db69c08c4ff7df6081499d20c7f544f91f167fd184cbf3487f1ced8f1e75c848
+  ✓ Transaction confirmed with 1 confirmations
+    Creating program account...
+    Program account created successfully
+    Deploying program transactions...
+Starting program deployment
+  ℹ ELF file size: 175640 bytes
+  → Deploying program with 210 transactions
+  Transactions sent successfully                                                                                                                                             ✓ Successfully sent 210 transactions for program deployment
+  [00:00:11] [########################################] 210/210 (100%)                                                                                                         Program transactions deployed successfully
+    Making program executable...
+    Transaction sent: 23de0a1d6fe6ec1f7304488aa223d092bcaa64ea63a23c61b765a248063e6e9c
+    Program made executable successfully
+  ✓ Program deployed successfully
+  ✓ Wallet 'devwallet' unloaded successfully.
+Your app has been deployed successfully!
+  ℹ Program ID: 3688ef8de06d56e32a765243e900875c4fefc6aa9c83dfbc2643f661c5b4982e
+  ```
+
+Copy the Program ID from the output as you will need this again later. 
+
+The Program ID can be thought of as a uniform resource locator (URL) for your deployed program on the Arch Network.
+
+### Create an Account
+An account is used to store the state for your dapp.
+
+Obtain the Program ID from the deployment step output and use it within this command; you may need to replace this if the Program ID you received from the previous step differs from this one.
+
+```bash
+arch-cli account create --name graffiti --program-id 3688ef8de06d56e32a765243e900875c4fefc6aa9c83dfbc2643f661c5b4982e
+```
+
+### Start the front-end application
+```bash
+arch-cli frontend start
```

Comment:
This is now demo and I think you address this in a later PR, but this command doesn't exist anymore.